src/__test__/backend/mcode/mock-data/index.js

```diff
@@ -26,7 +26,7 @@ export const similarPatientTreatmentsData = [
         "id": "row_2",
         "displayName": "hormonal therapy",
         "totalPatients": 12,
-        "survivalYears":[2,0,2,0,6,0],
+        "deathsPerYear":[2,0,2,0,6,0],
```

Comment:
I think I may have missed this in previous review.  But is this stating that what has been the number of patients who have survived in the past has now been replaced by the number who have died?  If so what was the reasoning behind this change?  I'm not real sure about it.

Reply:
Based on my assessment of how this data gets used, yes that's how this array gets used.